sdks/python/apache_beam/transforms/fully_qualified_named_transform_test.py

```diff
@@ -116,6 +119,14 @@ def test_glob_filter(self):
       with self.assertRaises(ValueError):
         FullyQualifiedNamedTransform._resolve('apache_beam.Row')
 
+  @patch('importlib.import_module')
+  def test_resolve(self, mock_import_module):
```

Comment:
Sorry to be a stickler, but this mocking doesn't actually convince me that the code in question actually does the right thing. Could you actually perform an import here so that we know we're interacting with the import module in the right way?

Reply:
Other tests already cover some real importing but I added two more that would have failed without the change in this PR.